app.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const app = express();
+const logger = require('./utils/loggers')
+app.use(express.json());
+
+const newsRoutes = require('./api/routes/news');
+
+//CORS Handling
+app.use((req,res,next) =>{
+    res.header('Access-Control-Allow-Origin','*');
+    res.header('Access-Control-Allow-Headers','Origin, Content-Type, Accept, Authorization')    
+    next();
+})
+
+//Logger to capture Api Endpoint, method and Timestamp
+app.use((req,res,next) => {
+    if((res.status(201)) || (res.status(200))){
+        logger.info(`Api EndPoint:${req.originalUrl}, Api Method: ${req.method}`, new Date())
+    } else {
+        logger.warn(`Error ${error.status} ` ,new Error(error.message));
+    }        
+    next();
+})
+
+//Routes to handle Request
+app.use('/news', newsRoutes)
+
+app.use((req,res,next)=> {
```

Comment:
Do we need that middleware? in case of we have error handler bellow

Reply:
I am using express router so this is how i understood.

Reply:
We can remove it

Reply:
Or better, lets replace with 

app.get('*', (req,res)=> {
  throw new CustomError('Not Found', 404)
})

Reply:
We should be good now